### PR TITLE
Test PHP 7.2

### PR DESCRIPTION
**Pull request by whikloj**
I thought because JsonLD relies on this (and I submitted https://github.com/lanthaler/JsonLD/pull/92) I should add a test for PHP 7.2 here as well.

That brought up some issues because Travis has changed and this repository seemed to fail tests that previously it passed.

I have done some moving of pieces to get all the test working again.

I didn't but I think you can remove [this line](https://github.com/lanthaler/IRI/blob/master/.travis.yml#L21) as there is no `http_test_server` installed so this does nothing.

I hope I haven't overstepped my bounds here.

cheers

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,8 +17,15 @@
...
         "lib-pcre": ">=4.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.8.*"
+    },
     "autoload": {
-        "psr-0": { "ML\\IRI": "" }
+        "classmap": ["src/"],
+        "psr-4": {
+            "ML\\IRI\\": "src/",
```
Comment (lanthaler) [inline composer.json]: Any reason to not simply point this to `""`, get rid of the second declaration and leave the `IRI.php` file where it currently is?

**Comment by lanthaler** [comment]
Thanks a lot for this Jared. Feel free to remove the `http_test_server` line as well, it was probably a copy-paste left-over

File: composer.json (hunk 1 above)
Comment (whikloj) [reply composer.json]: Fair point @lanthaler, this is the standard I've used and I get scared to deviate but it doesn't seem like its necessary.

**Comment by whikloj** [comment]
I realize the code-coverage is wrong now, I'll figure that out and add another commit here.

**Comment by lanthaler** [comment]
Thanks for these changes Jared

**Pull request merged**